src/app/[locale]/page.tsx

```diff
@@ -12,19 +19,41 @@ const COMPONENTS = [
   { id: 1, component: <Categories /> },
   { id: 2, component: <Advantages /> },
   { id: 3, component: <AboutApp /> },
-  { id: 4, component: <ExpertsFeed /> },
+  {
+    id: 4,
+    component: (top_experts: UserSearchOut[]) => (
+      <ExpertsFeed>
+        <ExpertsCarousel experts={top_experts} />
+      </ExpertsFeed>
+    ),
+  },
   { id: 5, component: <HowToStart /> },
   { id: 6, component: <Faq /> },
 ];
 
-export default function Home() {
+export default async function Home() {
+  const { aPIGetPublicJobStatistics } = getJobs();
+  const { aPIGetPublicTopExperts } = getUsers();
+  const {
+    data: { statistics },
+  } = await aPIGetPublicJobStatistics(backendURL);
+
+  const {
+    data: { top_experts },
+  } = await aPIGetPublicTopExperts({}, backendURL);
```

Comment:
Why do we need to do this fetch here? why cannot `ExpertsCarousel` do it inside itself?

Reply:
Because **ExpertsCarousel** is a client component. It is better to take the date from the server one.

Reply:
Then I suggest create a wrapper for the client component and do data fetching there and suspend wrapped component. That way, the page will be delivered to user faster